cli/cmd/volume.go

```diff
@@ -75,8 +83,29 @@ var volumeCreateCmd = &cobra.Command{
 	Short: helpVolumeCreateCmd,
 	Run: func(cmd *cobra.Command, args []string) {
 		validateNArgs(cmd, 2, 0)
-		volname := cmd.Flags().Args()[0]
-		bricks := cmd.Flags().Args()[1:]
+		force := cmd.Flags().Args()[0]
+		volname := cmd.Flags().Args()[1]
+		bricks := cmd.Flags().Args()[2:]
+		force_, err := strconv.ParseBool(force)
+		if err != nil {
+			fmt.Println("failed to parse boolean")
+			os.Exit(-1)
+		}
+		// vol, err := client.VolumeCreate(volname, bricks)
+		vol, err := client.VolumeCreate(api.VolCreateReq{
+			Name: volname,
+			Bricks: bricks,
+			Force: force_,
+		})
+		if err != nil {
+			log.WithField("volume", volname).Println("volume creation failed")
+			failure(fmt.Sprintf("Volume creation failed with %s", err.Error()), 1)
+		}
+		fmt.Println("Volume created successfully\n")
+		table := tablewriter.NewWriter(os.Stdout)
```

Comment:
Try to follow the same format as existing `gluster volume info` command. Table may not be suitable here.